draft-ietf-masque-connect-ip.md

```diff
@@ -833,9 +874,9 @@ Capsule Types" registry created by {{HTTP-DGRAM}}:
 
 |  Value   |        Type         |     Description     |   Reference   |
 |:---------|---------------------|:--------------------|:--------------|
-| 0xfff100 |   ADDRESS_ASSIGN    | Address Assignment  | This Document |
-| 0xfff101 |   ADDRESS_REQUEST   | Address Request     | This Document |
-| 0xfff102 | ROUTE_ADVERTISEMENT | Route Advertisement | This Document |
+| 0xfff200 |   ADDRESS_ASSIGN    | Address Assignment  | This Document |
+| 0xfff201 |   ADDRESS_REQUEST   | Address Request     | This Document |
```

Comment:
you had an opportunity to pick something using rand(), but you didn't take it?

Reply:
Fixed